imagetest/test_suites/metadata/setup.go

```diff
@@ -56,63 +205,54 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 	if err != nil {
 		return err
 	}
-	vm2.SetShutdownScript(shutdownScript)
+	for _, ms := range metadataScripts {
```

Comment:
the slice has a lot of keys for linux and windows, is it safe to add them all to a given vm? are you relying on some overwrite behavior, ordering, or client filtering?

Reply:
Yes, it's safe to add all of them to a given VM. The non-applicable metadata keys get ignored by the VM and if adding something like "windows-starutp-script-ps1" caused problems on Linux, I think we'd want to know about that.